lms/envs/common.py

```diff
@@ -2803,7 +2803,7 @@
 ANALYTICS_DATA_URL = ""
 ANALYTICS_DATA_TOKEN = ""
 ANALYTICS_DASHBOARD_URL = ""
-ANALYTICS_DASHBOARD_NAME = PLATFORM_NAME + " Insights"
+ANALYTICS_DASHBOARD_NAME = _('edX Insights')
```

Comment:
I think this should be 
```python
ANALYTICS_DASHBOARD_NAME = _(PLATFORM_NAME + ' Insights')
```

I have an instance running under "Client Insights" which wouldn't be the case if we did this.